x-pack/plugins/observability/server/assets/component_templates/slo_mappings_template.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const sloMappingsTemplate = {
+  template: {
+    mappings: {
+      properties: {
+        '@timestamp': {
+          type: 'date',
+          format: 'date_optional_time||epoch_millis',
+        },
+        slo: {
+          properties: {
+            id: {
+              type: 'keyword',
+              ignore_above: 256,
+            },
+            numerator: {
+              type: 'long',
+            },
+            denominator: {
+              type: 'long',
+            },
+            context: {
+              type: 'flattened',
+            },
+          },
+        },
+      },
+    },
+  },
+  _meta: {
+    description: 'Mappings for SLO rollup data',
```

Comment:
Let's add a `version` here... just use `1` for now. Then we can increment it every time we make a mapping change.